packages/ella/src/utils/constants/index.ts

```diff
@@ -24,3 +24,9 @@ export const CUSTOM_EVENTS = {
   TABLE_CLOSE: 'tableClose',
   TRACK_LOG: 'trackLog'
 };
+
+
+export const BROWSER_ORIGIN = {
```

Comment:
Is the name Browser origin correct here?

Msite and Desktop can be termed as platform